Use textarea for Verification link

edent opened the pull request:
Fixes #19965 

I think I've done this right!

edent committed:
Use textarea for Verification link

Fixes #19965 

I think I've done this right!

afontenot commented:
I see a couple issues with this implementation.

1. It needs `resize: none` set as a CSS style or else it looks weird. (You also didn't fix up the CSS; the existing input field has `border: 0` set, for example, matching on the tag name.)
2. It's broken if you have a long link. The height of the textarea needs to be set dynamically based on the content.

![example](https://user-images.githubusercontent.com/3870410/200283551-a1eaf490-34a9-40ae-a4f0-f5e4da802c9b.png)

edent commented:
Thanks for that.  I think manually setting the height to, say 5 rows should be enough for even the longest name and server. That would remove the need to dynamically calculate the height (difficult because we don't know how browsers will break lines.

I agree that removing the resize is sensible.

I don't understand what you mean by the CSS border. It might be my monitor, but I can't see any difference with the `<input>` and `<textarea>`

edent committed:
Improve rendering

Manually set the height to 5 characters, and remove the ability to resize the text area.

mjankowski commented:
Closing due to lack of activity and merge conflicts -- and lack of agreement on the linked issue.

If you feel strongly that this should be reconsidered, can you rebase/reopen and add more commentary on the approach?
